core/init.lua

```diff
@@ -207,6 +207,32 @@ function addon:UpdateButtonHighlight()
   end
 end
 
+
+function addon:AtSendMail()   
```

Comment:
Can you move these out of the addon space and either into their own, new, module, or some existing module?

Do the same for all these new addon methods.

Thanks!

Reply:
I can create a new module MovementFlow, do you want it in core or util folder ?

Reply:
Let's do util. Thanks!